setup.py

```diff
@@ -22,81 +22,60 @@ def read(*filenames):
         return f.read()
 
 
-def alltests():
-    import sys
-    import unittest
-    # use the zope.testrunner machinery to find all the
-    # test suites we've put under ourselves
-    import zope.testrunner.find
-    import zope.testrunner.options
-    here = os.path.abspath(os.path.join(os.path.dirname(__file__), 'src'))
-    args = sys.argv[:]
-    defaults = ["--test-path", here]
-    options = zope.testrunner.options.get_options(args, defaults)
-    suites = list(zope.testrunner.find.find_suites(options))
-    return unittest.TestSuite(suites)
-
-TESTS_REQUIRE = [
-    'zope.pagetemplate',
-    'zope.testing',
-    'zope.testrunner',
-]
+TESTS_REQUIRE = ["zope.pagetemplate", "zope.testing", "zope.testrunner"]
```

Comment:
No need to define this globally if you're dropping `tests_require=...` and using it in only one place.